### DISCUSSION TITLE
Install and Deploy Nodejs Apps On AIX

### DISCUSSION THREAD

**Discussion by 3mrhussein** [nodejs/help#4734] [Q&A]
I've searched for a practice of installing Nodejs and deploying Express Apps on AIX operating system but with no luck. (found Nothing)
I hope if someone have experienced this could share his practice here in this Discussion

**Comment by bnoordhuis** [comment]
cc @nodejs/platform-aix

**Comment by aixtools** [comment]
I wish I could help on this - but I only do sysadmin for AIX OSS projects. I have not used nodejs myself.

**Comment by mhdawson** [comment]
@3amr7ussein are you running into some issues as the goal is for Node.js to run in the same way that it does on other platforms. One thing I do know is that if you want to use file watching then AHAFS must be enable which is outlined in https://nodejs.org/docs/latest/api/fs.html#fswatchfilename-options-listener

**Reply by 3mrhussein** [reply]
Acutally it is my first time to deal with AIX so I don't have any experince with AIX OS at all, searching for practices with NodeJs on AIX and found nothing, make me feel like it is a bad decision trying to run Nodejs service on AIX 🫣
So Iam asking to get feedback from developers who already practice

**Reply by mhdawson** [reply]
@3amr7ussein I don't have any names of developers I can pass on here, but will confirm that there are people within IBM and Red Hat that work to keep Node.js running on AIX and that the community project publishes AIX binaries.  If you are an IBM customer if you want to email me (see email on main Node.js README.md) I can possibly connect you to people in the AIX/systems team that might be able to. 

The main thing is that there should not need to be much AIX specific docs/blogs/etc. as running Node.js there should mostly be the same as on other unix variants.

**Reply by 3mrhussein** [reply]
@mhdawson Thanks Micheal for your reply, I'll contact you soon if you don't mind, and yes I'm an IBM Customer, will try first to deploy it myself and see what things will lead to , anyway I'll upgrade the status on this Discussion.
